engine/src/main/java/com/datatorrent/stram/client/StramAgent.java

```
@@ -424,7 +433,7 @@ private StramWebServicesInfo retrieveWebServicesInfo(String appId)
       return new StramWebServicesInfo(appMasterUrl, version, appPath, user, secToken, permissionsInfo);
     }
     catch (Exception ex) {
-      LOG.debug("Caught exception when retrieving web service info for app " + appId, ex);
+      LOG.warn("Caught exception when retrieving web service info for app {}", appId, ex);
       return null;
```

Exceptions should be propagated instead of logging.